components/efuse/src/esp_efuse_utility.c

```diff
@@ -15,6 +15,8 @@
 
 static const char *TAG = "efuse";
 
+static const char *TAGV = "efuse (virtual)";
```

Comment:
I was actually suggesting something like:

```
#if CONFIG_EFUSE_VIRTUAL
static const char *TAG = "efuse (virtual)";
#else
static const char *TAG = "efuse";
#endif
```

CC @KonstantinKondrashov

Reply:
I like that suggestion. Updated the PR.